lib/vdsm/storage/lvm.py

```diff
@@ -1439,6 +1439,26 @@ def createVG(vgName, devices, initialTag, metadataSize, force=False):
         _lvminfo._invalidatevgs(vgName)
         log.debug("Cache after createvg %s", _lvminfo._vgs)
 
+    # Disable autoactivation after creating a VG
+    _disable_autoactivation_vg(vgName)
+
+
+def _disable_autoactivation_vg(vg_name):
+    """Avoid undesired VG activation by disabling autoactivation attribute
+    on a given VG.
```

Comment:
We always use this style:

    """
    Title ...

We need more info here on how disabling autoactivation works. How does it 
protect from activation? Which lvm command does autoactivation?

I think adding a test like this will make it more clear:
1. create and configure a new vg
2. create some lvs
3. run lvm commands that does auto activation
4. verify that no lv was activated

Other tests creating and activating lvs already verify lv activation flows.

Reply:
Ok, I'll mark the PR as draft for now and do some checks to see how can I test this.

I'll re-request the review when ready :)